crates/scheduler/src/call/crypto/hash.rs

```diff
@@ -88,7 +99,6 @@ fn update<B: Board>(mut call: SchedulerCall<B, api::update::Sig>) {
             HashContext::Sha384(context) => context.update(data),
             _ => trap_use!(data),
         }
-        Ok(())
     };
     call.reply(result);
```

Comment:
For the inference problem you get here, you have to annotate the result variable:

```rust
let result: Result<Result<(), _>, _> = try {
```

(some for the other occurrence below)